README.rst

```diff
@@ -1144,8 +1144,35 @@ When operating on stdin, use the ``--input-type`` and ``--output-type`` flags as
 
     $ cat myfile.json | sops --input-type json --output-type json -d /dev/stdin
 
+JSON indentation
+~~~~~~~~~~~~~~~~
+
+``sops`` indent ``SOPS`` files by default using one ``tab``. However, you can change
+this default behaviour to use ``spaces`` by either using the additional ``--indent=2`` CLI option or
+by configuring ``.sops.yaml`` with the code below.
+
+The special value ``0`` disables indentation, and ``-1`` uses a single tab.
+
+.. code:: yaml
```

Comment:
Without that newline, all the content are parameters and the directive has no content, thus showing an empty code listing.
```suggestion
.. code:: yaml

```